apps/participants/models.py

```diff
@@ -7,7 +7,9 @@
 
 
 class Participant(TimeStampedModel):
-
+    """
+    Model representing the Participant of the competition. 
+    """
```

Comment:
@souravsingh if the docstring is of one line, then it should be like this 
"""Model representing the Participant of the competition. """ according to the PEP8 guidelines.